models/renalStructures_CECT_segmentation/docs/README.md

```diff
@@ -0,0 +1,122 @@
+# Model Title
+Renal structures CECT segmentation
+
+### **Authors**
+Ivan Chernenkiy,   Michael Chernenkiy,   Dmitry Fiev,   Evgeny Sirota, Center for Neural Network Technologies / Institute of Urology and Human Reproductive Systems / Sechenov First Moscow State Medical University
+
+### **Tags**
+Segmentation, CT, CECT, Kidney, Renal, Supervised
+
+## **Model Description**
+The model is the SegResNet architecture[1] for volumetric (3D) renal structures segmentation. Input is artery, vein, excretory phases after mutual registration and concatenated to 3 channel 3D tensor.
+
+
+## **Data**
+DICOM data from 41 patients with kidney neoplasms were used. Data included all phases of contrast-enhanced multispiral computed tomography. We split the data: 32 observations for the training set and 9 – for the validation set. At the labeling stage, the arterial, venous, and excretory phases were taken, affine registration was performed to jointly match the location of the kidneys, and noise was removed using a median filter and a non-local means filter. Validation set ip published to Yandex.Disk. You can download via [link](https://disk.yandex.ru/d/pWEKt6D3qi3-aw) or use following command:
```

Comment:
Hi @blacky-i , thanks for the contribution. As for the dataset this bundle used, could you please also mention the related license?
see:
https://github.com/Project-MONAI/model-zoo/blob/b245833e22544126f23925eac89da471562467a1/CONTRIBUTING.md#license-format

Here is an example:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_axial_slices_generative_diffusion/docs/data_license.txt

Reply:
Hello, added license information to `docs/` folder